.github/workflows/aa_use_case_test.yml

```diff
@@ -336,6 +336,86 @@ jobs:
           name: testlog_${{github.job}}
           path: |
             ${{github.workspace}}/developer/aa_test.log
+  
+  aatest_ds000114_fmri_symlinks:
+    name: Testing fMRI with symlinks
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@master
+        with:
+          root-reserve-mb: ${{ env.DISKSIZE }}
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+          remove-android: 'true'
+          remove-haskell: 'true'
+
+      - name: Set git to use LF
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+
+      - name: Check out repository under $GITHUB_WORKSPACE
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          # Semantic version range syntax or exact version of a Python version
+          python-version: '3.6'
+          # Optional - x64 or x86 architecture, defaults to x64
+          architecture: 'x64'
+
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v1
+        with:
+          # MATLAB release to set up R2020a
+          release: R2020a
+
+      - name: Install tools and configure aa
+        run: |
+          python trigger_install.py
+        working-directory: ${{github.workspace}}/.github/workflows
+        env:
+          LOAD_FSL: 0
+          LOAD_FREESURFER: 0
+          PARAMETER_XML: aap_parameters_defaults_GitHub.xml
+
+      - name: Retrive test data under $HOME/data/aa_demo
+        run: |
+          mkdir -p $HOME/data/aa_demo
+          aws s3 cp s3://openneuro.org/ds000114 $HOME/data/aa_demo/ds000114 --quiet --recursive --exclude "*" --include "task-*" --no-sign-request
+          aws s3 cp s3://openneuro.org/ds000114 $HOME/data/aa_demo/ds000114 --quiet --recursive --exclude "*" --include "dwi*" --no-sign-request
+          aws s3 cp s3://openneuro.org/ds000114/sub-01 $HOME/data/aa_demo/ds000114/sub-01 --quiet --recursive --no-sign-request
+          aws s3 cp s3://openneuro.org/ds000114/sub-02 $HOME/data/aa_demo/ds000114/sub-02 --quiet --recursive --no-sign-request
+          aws s3 cp s3://openneuro.org/ds000114/sub-03 $HOME/data/aa_demo/ds000114/sub-03 --quiet --recursive --no-sign-request
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_EC2_METADATA_DISABLED: true
+
+      - name: Run script
+        uses: matlab-actions/run-command@v1
+        with:
+          command: addpath(getenv('GITHUB_WORKSPACE')); aa_ver5; SPM = spmClass(fullfile(getenv('HOME'),'tools','spm12')); SPM.load; aa_test('glob','aatest_ds000114_fmri','haltonerror',true)
```

Comment:
In this CI job, you use the original `aatest_ds000114_fmri` rather than your `aatest_ds000114_fmri_symlinks` script.

Reply:
> In this CI job, you use the original `aatest_ds000114_fmri` rather than your `aatest_ds000114_fmri_symlinks` script.

How can I set the symlinks option then? Shall I make a new parameter file for it?

Reply:
In the `aa_test` call, you should refer to the script you created:
`aa_test('glob','aatest_ds000114_fmri_symlinks','haltonerror',true)`